Apps/Common/src/AspNetConfiguration/StartupConfiguration.cs

```diff
@@ -307,6 +312,100 @@ public void ConfigureAuthServicesForJwtBearer(IServiceCollection services)
             });
         }
 
+        /// <summary>
+        /// This sets up the OIDC authentication.
+        /// </summary>
+        /// <param name="services">The passed in IServiceCollection.</param>
+        public void ConfigureOpenIdConnectServices(IServiceCollection services)
```

Comment:
this method seems like is only used in Jobscheduler.  and has hardcoded cookie names for said project. If not used anywhere else, lets move it out of common

Reply:
Forgot to refactor that.  The plan is to use this common across JobScheduler and AdminWebClient. Will refactor.